frontend/src/app/home/common-news/common-news.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="app-common-news">
+  <input type="radio" checked="checked" id="slide1" class="app-common-news__selector" name="app-common-news__selector" />
```

Comment:
What if in the future you need to add six news instead of three? :)
I think it would be better to use cycle which dependent on the news count